### PR TITLE
Web provider tick

### PR DESCRIPTION
**Pull request by Adam-Goldman7**
@tomusdrw  if u think it's a good direction, I'll add some tests to it

### CODE REVIEW

File: client/provider/web3-provider.js [hunk 1]
```diff
@@ -15,24 +16,22 @@ export class Web3Provider extends Web3Base {
...
-      toPromise(this.ethcoreWeb3.getDevLogsLevels).then(DebugActions.updateDevLogsLevels),
-      toPromise(this.ethcoreWeb3.getDevLogs).then(DebugActions.updateDevLogs)
-    ])
+    return Promise.map(this.tickArr, obj => {
+      return toPromise(obj.method).then(obj.actionMaker)
+        .catch(err => {
+          this.store.dispatch(StatusActions.error(err));
+          return false; // don't process errors in the promise chain
+        });
+    })
```
Comment (Adam-Goldman7) [inline client/provider/web3-provider.js]: mayb there's a more elegant solution to this?

or we'll create one :)

File: client/reducers/status.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ export default handleActions({
   error (state, action) {
     return {
       ...state,
-      error: action.payload,
+      disconnected: (action.payload.message === 'Invalid JSON RPC response: ""'),
```
Comment (tomusdrw) [inline client/reducers/status.js]: Is there no better way to detect disconnection?

Comment (tomusdrw) [inline client/reducers/status.js]: We should somehow display other errors too - now we just ignore them.

File: client/provider/web3-provider.js (hunk 1 above)
Comment (tomusdrw) [inline client/provider/web3-provider.js]: I think we don't need `Promise.map`. Wouldn't that work exactly the same?
```javascript
return Promise.all(this.tickArr.map(obj => {
      return toPromise(obj.method).then(obj.actionMaker)
        .catch(err => {
          this.store.dispatch(StatusActions.error(err));
          return false; // don't process errors in the promise chain
        });
    })
);
```

File: client/reducers/status.js (hunk 2 above)
Comment (Adam-Goldman7) [inline client/reducers/status.js]: Maybe ...
we'll put a small icon in the header the is linkable to "debug" page?

displaying a toastr/popup or something similar on every error will be too intrusive I thnk

Comment (Adam-Goldman7) [inline client/reducers/status.js]: I haven't seen a better way to detect disconnection. u have an idea?

Comment (tomusdrw) [inline client/reducers/status.js]: 1. I think just an indication that there were some errors would be good (like a yellow warning sign with tooltip: "there were errors: check the console"
2. I think there is none. But we should check for `CONNECTION ERROR` not invalid rpc response.

**Pull request merged**